auth-api/src/auth_api/utils/enums.py

```diff
@@ -200,6 +200,13 @@ class InvitationType(Enum):
     STANDARD = 'STANDARD'  # Used to indicate the standard email invite with admin approval
 
 
+class AffiliationInvitationType(Enum):
+    """Affiliation Invitation type."""
+
+    EMAIL = 'EMAIL'  # Used to indicate an affiliation invitation initiated through email
```

Comment:
comment seems redundant? (just my opinion, could ignore this)

Reply:
Removed comment